### ISSUE TITLE
Slit-like cerebral ventricles

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10585]
**Preferred term label:**
Slit-like cerebral ventricles

**Synonyms**
small, reduced volume

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/browse/term/HP:0002118

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.omim.org/entry/620746 "MRI findings included slit-like lateral ventricles in all patients"

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000989
- label: Slit-like lateral ventricle
- parents: [Abnormal lateral ventricle morphology (HP:0030047)](https://hpo.jax.org/app/browse/term/HP:0030047)
- pmid: PMID:38054405
- def: Abnormally small lateral ventricle with a slit-like appearance.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10585
- synonyms:
